packages/salesforcedx-vscode-core/test/vscode-integration/orgBrowser/metadataCmp.test.ts

```diff
@@ -129,15 +129,9 @@ describe('build metadata components list', () => {
     }
   });
 
-  it('should only return "unmanaged" components if they have a manageableState', async () => {
+  it('should not return components if they are uneditable', async () => {
     const type = 'ApexClass';
-    const states = [
-      'unmanaged',
-      'installed',
-      'released',
-      'deleted',
-      'depricated'
-    ];
+    const states = ['installed', 'released', 'deleted', 'deprecated'];
```

Comment:
@AnanyaJha these are the other states and they are managed package states, which are not editable components. I should add `beta` to this test as well.